include/pfs/drp/stella/SpectraTemplates.hpp

```diff
@@ -426,9 +441,11 @@ bool pfs::drp::stella::Spectrum<SpectrumT, MaskT, VarianceT, WavelengthT>::ident
       indices.erase( indices.begin() + randNum );
     }
 
-    if ( nInd < ( std::round( double( lineList.getShape()[ 0 ] ) * 0.66 ) ) ){
-      std::string message("pfs::drp::stella::identify: ERROR: ");
-      message += "identify: ERROR: less than " + std::to_string( std::round( double( lineList.getShape()[ 0 ] ) * 0.66 ) ) + " lines identified";
+    _nGoodLines = nInd;
+    const long nLinesIdentifiedMin(std::lround(double(lineList.getShape()[0]) * 0.66));
```

Comment:
Replace with a parameter I added to DispCorControl. This is the minimum ratio of lines in the line list which needs to be identified for <identify> to pass